packages/client/css/generic/message.css

```diff
@@ -41,6 +41,10 @@
   vertical-align: -.4em;
 }
 
+.Message.\--big-emotes .Message-emote {
+  width: 4rem;
```

Comment:
Wouldn't it make more sense to use `em` here, so that if we change the font size of the messages, the size of emojis changes too?

Reply:
Yeah. Normal emotes use `rem` too, however, so this is probably out-of-scope for this PR.